hist/histv7/inc/ROOT/RLinearizedIndex.hxx

```diff
@@ -0,0 +1,22 @@
+#ifndef ROOT_RLinearizedIndex
+#define ROOT_RLinearizedIndex
+
+#include <cstddef>
+
+namespace ROOT {
+namespace Experimental {
+
+/**
+A linearized index that can be invalid.
+
+For example, when an argument is outside the axis and underflow / overflow bins are disabled.
+*/
+struct RLinearizedIndex final {
+   std::size_t index;
+   bool valid;
```

Comment:
Style: `fIndex`, `fValid`

Reply:
Ok, I wasn't sure if we want to follow the style here or keep it closer to STL types (`std::pair` with `first` and `second), but I guess uniformity in ROOT wins.